<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serail Test</string>
  </property>
  <widget class="QPushButton" name="giveInstantly">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>right now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setNextTime">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>setNextTime</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="hour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>set Next Hour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>260</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>set Next Minute</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minute">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>300</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>set Next Second</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="second">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Left Time :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondlabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="hourlabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="minutelabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Manjari</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Voice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="btn_rec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>REC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
